extension/sqlsmith/fuzzyduck.cpp

```diff
@@ -62,18 +62,25 @@ void FuzzyDuck::FuzzAllFunctions() {
 	EndFuzzing();
 }
 
-string FuzzyDuck::GenerateQuery() {
-	LogTask("Generating query with seed " + to_string(seed));
+idx_t FuzzyDuck::GenerateRandomNuber() {
 	auto &engine = RandomEngine::Get(context);
-	// set the seed
 	engine.SetSeed(seed);
-	// get the next seed
 	seed = engine.NextRandomInteger();
+	return seed;
+}
+
+string FuzzyDuck::GenerateQuery(idx_t number_of_statements) {
+	LogTask("Generating query with seed " + to_string(seed));
+	seed = GenerateRandomNuber();
 
 	// generate the statement
 	StatementGenerator generator(context);
-	auto statement = generator.GenerateStatement();
-	return statement->ToString();
+	// accumulate statement(s)
+	string statement = "";
+	for (size_t i = 0; i < (size_t)number_of_statements; i++) {
```

Comment:
idx_t is usually preferred at DuckDB

Reply:
Do you mean instead of the `string`?

Reply:
Changed it to idx_t in the loop and took to the branch `multi-statement`